oak_loader/src/options.rs

```diff
@@ -326,6 +332,23 @@ fn get_root_tls_certificate_or_default(opt: &Opt) -> anyhow::Result<String> {
     }
 }
 
+/// Similar to `get_root_tls_certificate_or_default`, but instead of returning the content of the
+/// certificate file, returns the file name.
+fn get_root_tls_certificate_path_or_default(opt: &Opt) -> anyhow::Result<String> {
+    match &opt.root_tls_certificate {
+        Some(certificate_path) => {
+            if !cfg!(feature = "oak_debug") {
+                // Unreachable: it should not be possible to specify the flag without `oak_debug`.
+                anyhow::bail!(
+                    "Specifying `root-tls-certificate` requires the `oak_debug` feature."
+                );
+            };
+            Ok(certificate_path.to_owned())
+        }
+        None => Ok("certs/roots.pem".to_string()),
```

Comment:
These root certs don't seem to end up being statically compiled in the executable binary, what stops someone from just replacing this file and pointing to an arbitrary self-signed certs at runtime? Is this also what we are doing for the gRPC client pseudo-node? (if so, I think that must be fixed there too)

cc @conradgrobler